.gitignore

```diff
@@ -68,4 +68,6 @@ transactionsDB/
 
 certificates/ca_bundle.crt
 certificates/certificate.crt
-certificates/private.key
\ No newline at end of file
+certificates/private.key
+
+.with.json
```

Comment:
What does this file do?

Reply:
holds the give parameters as json.

A package script like "npm run terminal" has other commands in it and we can't pass arguments dynamically to those commands.

So we needed a way to load up those given arguments in other given commands.

npm run with server:domain=google.com has as purpose just to parse those arguments and save them to that file.
npm run commands checks to see if that file exists. If it exists and it has data in it, then it gets loaded. If there are arguments that are in given defaults (as i stated above) then they get overwritten by the ones from the file.